e2e/tests/datasets/export-to-notebook.spec.ts

```diff
@@ -0,0 +1,53 @@
+import {findWorkspace, signIn, waitWhileLoading} from 'utils/test-utils';
+import DataPage from 'app/page/data-page';
+import DatasetSaveModal from 'app/page/dataset-save-modal';
+import {makeRandomName} from 'utils/str-utils';
+import NotebookPreviewPage from 'app/page/notebook-preview-page';
+
+describe('Create Dataset', () => {
+
+  beforeEach(async () => {
+    await signIn(page);
+  });
+
+   /**
+    * Create new Dataset, export to notebook in Python language
+    * Finally delete Dataset.
+    */
+  test('Export dataset to notebook in Python programming language', async () => {
+    const workspaceCard = await findWorkspace(page);
+    await workspaceCard.clickWorkspaceName();
+
+      // Click Add Datasets button
```

Comment:
nit: fix indentation for all comments in this file. These shouldn't be indented deeper than the code